### ISSUE TITLE
Toilets with drinking water are being shown as drinking water with toilets

### ISSUE THREAD

**Issue by osmuser63783** [organicmaps/organicmaps#7431]
**Describe the issue**
When public toilets have drinking water, this is tagged as [`drinking_water=yes`](https://wiki.openstreetmap.org/wiki/Key:drinking_water). This may mean that there is a water fountain, or it may merely mean that the water from the tap where people wash their hands is fit for human consumption.

When an object has `amenity=toilets` and `drinking_water=yes`, in Organic Maps, the drinking water takes precedence: the icon shown on the map is the drinking water icon (a drop of water). When you select it, it says "Drinking Water" and there is a little icon to show that the object has toilets, like when a restaurant has toilets.

In other words, it gets shown to the user as a water fountain with toilets, when in reality it is public toilets with drinking water.

**Steps to reproduce**
1. Find any `amenity=toilets` with `drinking_water=yes` ([query](https://overpass-ultra.us/#m=13.67/41.9027/12.4694&q=LQhQGcBcE8BsFMBcACAFpSAHciD0uo55wA6SAJwEMBLWKSk8VXAe03gDsBbSzSW4sBbguJAFbgWHUCGChcAKlAAVVNXDJ1ySh2TwAHpS6YEyAPIA3eOUyVwGgI4BXa9BIry0TZGQsnPgCMvTHJicGoOAHNkSFR4ZAAlJ10A-0gpbQCWKwBCUAVcUABtAKz9RABvCtKWfQBfOoBdAG5QAApQDgB3ciKjTmoYAF4AInSBSHARxqKAE3IIgGsIyIB9LspIayHoYhbQAEpWvx9I+BYuZqA))
2. Select it

**Expected behaviour**
I would expect an object with `amenity=toilets` and `drinking_water=yes` to render on the map like public toilets. The toilets are the "main tag" (amenity) and the object will look like a public toilet from the outside.

When you select the toilets, the "title" should be "Toilet", not "Drinking Water". It could show a little icon that makes it clear the water there is drinkable.

The toilets could show up both when searching for toilets and when searching for drinking water, but they should show up in the list of search results as "Toilet" not as "Drinking Water".

**Screenshots**

**System information:**
 - Organic Maps version: 2024.02.06-11

**Comment by kari909** [comment]
Yes. That is the problem. I think it’s better to remove such marks from toilets

**Comment by vbertola** [comment]
I came here to report the same thing, which I just noticed in the map for this object in San Francisco:
https://www.openstreetmap.org/way/34218909
The app does not show the "toilets" POI icon, but just a building with the drinking water icon.

**Comment by dvdmrtnz** [comment]
A drinking_water-yes type should be added, same as it was done in https://github.com/organicmaps/organicmaps/pull/7108 with toilets-yes

**Comment by osmuser63783** [comment]
Amazing, thanks for fixing it so quickly!
